src/Informer/Verbosity.hpp

```diff
@@ -0,0 +1,32 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <ostream>
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+
+/// \cond
+template <typename T> struct create_from_yaml;
+class Option;
+/// \endcond
+
+/// Indicates how much informative output a class should output.
+enum class Verbosity { Silent, Quiet, Verbose, Debug };
+
+namespace Tags {
```

Comment:
should this be a CacheTag instead of a DataBox tag?

Reply:
Also move the tag to a separate file Tags.hpp

Reply:
I put it in the DataBox inside the AH actions.  So doesn't it need to be a DataBox tag?

Reply:
Sorry, that was more of a design question.  In other words, isn't this a constant and therefore could it not be stored in the ConstGlobalCache instead?

Reply:
If I do that, can I have different verbosities for AhA and AhC (e.g. make AhA quiet and AhC verbose)?  And a different verbosity for the Interpolator?